programs/fileio.c

```diff
@@ -814,6 +814,7 @@ static void FIO_adjustParamsForPatchFromMode(FIO_prefs_t* const prefs,
     }
     if (cParams.strategy >= ZSTD_btopt) {
         DISPLAYLEVEL(1, "[Optimal parser notes] Consider the following to improve patch size at the cost of speed:\n");
+		DISPLAYLEVEL(1, "- Use --single-thread mode in the zstd cli");
```

Comment:
nit (very minor) : alignment seems messed up due to use of `tab` instead of `space`

Reply:
Ah yes good call. Switched to vim to make that quick change. I'll fix it